lib/support/cli.js

```diff
@@ -333,7 +333,7 @@ module.exports.parseCommandLine = function parseCommandLine() {
     })
     */
     .option('outputFolder', {
-      describe: 'The folder name where the result will be stored. By default the name is generated using current_date/domain/filename',
+      describe: 'The folder where the result will be stored.',
       default: 'sitespeed-result'
```

Comment:
ooops, we should have deleted the default, this got out in the latest beta and breaks the output dir behaviour :(